templates/adspot/Inspector.tsx

```
@@ -0,0 +1,152 @@
+'use client'
+import { Button, Input, Select, RadioGroup, Label, ColorPicker } from '@/sdk/components'
+import { useFrameConfig, useFarcasterId, useUploadImage } from '@/sdk/hooks'
+import { Configuration } from '@/sdk/inspector'
+import React from 'react'
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)